.github/workflows/build-deploy-book.yml

```diff
@@ -0,0 +1,59 @@
+name: build-deploy-book
+
+# Only run this when the master branch changes
+on:
+  push:
+    branches:
+    - utf8
+  pull_request:
+    branches:
+    - utf8
+
+# This job installs dependencies, build the book, and pushes it to `gh-pages`
+jobs:
+  build-deploy-book:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+    steps:
+    - uses: actions/checkout@v2
+
+    # Install dependencies
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    # Set up Miniconda with environment
+    - name: Setup Miniconda
+      uses: goanpeca/setup-miniconda@v1.1.2
+      with:
+        environment-file: windows_tests/environment.yml
+        activate-environment: buildit
+        python-version: 3.7
+        auto-activate-base: false
+
+    # Build the book
+    - name: Build the book
+      shell: bash -l {0}
+      run: |
+        pip install -e . --upgrade
+        ORIG_DIR="${PWD}"
+        mkdir -p html_out
+        cd windows_tests/quantecon_flat/mini_book
+        jupyter-book build docs
+        echo "changing to ${ORIG_DIR}/html_out/flat_test"
+        cp -a docs/_build/html "${ORIG_DIR}/html_out/flat_test"
+        cd "${ORIG_DIR}/html_out/flat_test"
+        echo "now in ${CWD}"
+        ls -lrtd *
+
+
+    # Push the book's HTML to github-pages
```

Comment:
is there a reason we want a windows-specific docs deploy?

Reply:
for the record, the motivation behind the complicated workflow is to allow me to debug separate book problems by rendering independent html trees to the same gh-pages.  The
environment takes about 4 minutes to set up for windows, so be able to do both the
encoding debugging and the unrelated nested/flat table of contents debugging (coming
next) in the same environment is a timesaver.